### PR TITLE
Chore/handle cached response

### PR DESCRIPTION
**Pull request by StepanovDaniilOlegovich**
## Description
It a small but crucial fix. Because of the wrong condition LastRequesteStorage fallback was not working properly and we need to set timestamp on first image request too.

## Motivation and Context
Bugfix

## How Has This Been Tested?
unit tests

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/store-files-mixin.js [hunk 1]
```diff
@@ -135,6 +135,11 @@ export const StoreFilesMixin = dedupingMixin( base => {
       return fetch( fileUrl )
         .then( resp => {
           respToCache = resp.clone();
+
+          let timestamp = new Date();
+
+          this.lastRequestedStorage.save( fileUrl, timestamp.toUTCString());
```
Comment (stulees) [inline src/store-files-mixin.js:141]: @StepanovDaniilOlegovich Why was this changed to store a UTC string and not a `getTime()` timestamp (number)? The eventual common-template script would be doing a comparison of current timestamp to the saved timestamp. What is the advantage of using UTC string in this comparison?

File: test/unit/store-files-mixin-last-requested.html [hunk 2]
```diff
@@ -117,9 +117,6 @@
 
         lastRequestedStorage.save("test");
         assert.isFalse(lastRequestedStorage._getMap.called);
-
-        lastRequestedStorage.save("test", "test");
```
Comment (stulees) [inline test/unit/store-files-mixin-last-requested.html:121]: @StepanovDaniilOlegovich Can you clarify why code was added but tests are being removed? Thanks

Comment (StepanovDaniilOlegovich) [reply test/unit/store-files-mixin-last-requested.html:121]: this test accounts for deleted condition

File: src/store-files-mixin.js (hunk 1 above)
Comment (StepanovDaniilOlegovich) [reply src/store-files-mixin.js:141]: @stulees  this is date format required by If-modified-since header

Comment (stulees) [reply src/store-files-mixin.js:141]: @StepanovDaniilOlegovich Okay thanks

**Comment by StepanovDaniilOlegovich** [comment]
@stulees ready for review
